cpp/test/phonenumbers/phonenumberutil_test.cc

```diff
@@ -1444,6 +1450,12 @@ TEST_F(PhoneNumberUtilTest, GetLengthOfNationalDestinationCode) {
   number.set_country_code(800);
   number.set_national_number(12345678ULL);
   EXPECT_EQ(4, phone_util_.GetLengthOfNationalDestinationCode(number));
+
+  // A mobile number from China is geographical, but does not have an area code.
```

Comment:
Move this test to the test method testing GetLengthOfGeographicalAreaCode :)

And add a test for GetLengthOfNationalDestinationCode for China with comment:

```
  // A mobile number from China is geographical, but does not have an area code:
  // however it still can be considered to have a national destination code.
```